testsuite/features/support/constants.rb

```diff
@@ -944,13 +940,23 @@
       sle-product-suse-manager-proxy-4.3-updates-x86_64
       sle-module-suse-manager-proxy-4.3-pool-x86_64
       sle-module-suse-manager-proxy-4.3-updates-x86_64
+      sle-module-basesystem15-sp4-pool-x86_64-proxy-4.3
```

Comment:
I'm pretty sure we don't need to explicitly mention basesystem here since it will mandatory channel for the product, but let's be explicit even if it's not needed.

Reply:
That's the thing!
This map contains all the channels that we need to wait to be synchronized.
Meaning, this map is not about triggering channels, but about waiting for those we expect to be synchronized.